.github/workflows/generate-preview.yml

```
@@ -29,6 +29,30 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
+      - name: Remove create-preview label
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.removeLabels({
+              issue_number: github.event.pull_request.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ['create-preview']
+            })
+
+      - name: Add generating-preview label
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.addLabels({
+              issue_number: github.event.pull_request.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ['create-preview']
```

I think the signature is a little different from the remove. If you decide to keep this step
```suggestion
              labels: ['create-preview']
```